src/main/java/offtop/UserService/listeners/UserDataConsumer.java

```diff
@@ -0,0 +1,33 @@
+package offtop.UserService.Services;
+
+import java.util.Map;
+
+import java.util.*;
+// import org.json.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+import offtop.UserService.Services.UserService;
+import offtop.UserService.Models.User;
+import offtop.UserService.Listeners.UserDataProducer;
+
+@Service
+public class UserDataConsumer {
+    private final Logger logger = LoggerFactory.getLogger(UserDataConsumer.class);
+
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    UserDataProducer userDataProducer;
+
+    @KafkaListener(topics = "requestAllUserData", groupId = "group_Id")
+    public void handleAllUsersDataRequest(String message) {
+      List<User> allUsers = userService.getAllUsers();
+      userDataProducer.sendAllUserData(allUsers);
+      logger.info(String.format("#### -> Consumed message -> %s", message));
```

Comment:
make the logger be on the first line of this function

Reply:
Otherwise, our server will log the data in an awkward order like this

![awk_log](https://user-images.githubusercontent.com/26909882/87004933-13742500-c173-11ea-8599-439a08009188.png)

Reply:
Because we should be logging our consumed message as soon as we subscribe to it